docs/install/amazon-web-services-eks.md

```diff
@@ -100,17 +100,17 @@ If you receive the message `No resources found` or see nodes marked as `NotReady
 
 ### Step 1 - Conducting Preflight Checks
 
-Run the following command to conduct preflight checks against the EKS cluster to ensure that Ondat prerequisites are in place before continuing with installation.
+- Run the following command to conduct preflight checks against the EKS cluster to ensure that Ondat prerequisites are in place before continuing with installation.
 
 ```bash
 kubectl storageos preflight
 ```
 
 ### Step 2 - Installing Ondat
 
-1. Define and export the `STORAGEOS_USERNAME` and `STORAGEOS_PASSWORD` environment variables that will be used to manage your Ondat instance. Set the `StorageClass` for etcd to use (this cannot be Ondat, as Ondat is dependent upon etcd). On AWS EKS, we suggest `gp3` for a good balance of performance and resilience or `io2` where top performance is essential.
-
-The default `StorageClass` in EKS is `gp2` which is not recommended, instead we will create a `gp3` `StorageClass` and set it as default, at least until we install Ondat:
+1. Define and export the `STORAGEOS_USERNAME` and `STORAGEOS_PASSWORD` environment variables that will be used to manage your Ondat instance. 
+2. Set the `StorageClass` for etcd to use (this cannot be Ondat, as Ondat is dependent upon etcd). On AWS EKS, we suggest `gp3` for a good balance of performance and resilience, or `io2` where top performance is essential.
```

Comment:
should we break out the "this cannot be ondat...." to make it a note with the emoji, like the rest of the paragraph ?

Reply:
yes - good idea - I have made the change